### PR TITLE
add GithubAccess btn to noNewNotifications cell

### PR DESCRIPTION
**Pull request by BrianLitwin**
Final: 

screenshot 2018-08-11 12 16 07

Thanks for feedback! 

quick edit: capitalized the A in "Review GitHub access" in 2nd commit

### CODE REVIEW

File: Classes/Notifications/NotificationsViewController.swift [hunk 1]
```diff
@@ -304,4 +307,13 @@ BaseListViewController2EmptyDataSource {
...
         didSingleTapTab()
     }
+    
+    // MARK: ReviewGithubAccessDelegate
+    func reviewGithubAccessButtonTapped() {
+        //copied/pasted from SettingsViewController .. could consolidate 
+        guard let url = URL(string: "https://github.com/settings/connections/applications/\(Secrets.GitHub.clientId)")
+            else { fatalError("Should always create GitHub issue URL") }
+        // iOS 11 login uses SFAuthenticationSession which shares credentials with Safari.app
+        UIApplication.shared.open(url, options: [:])
```
Comment (BasThomas) [inline Classes/Notifications/NotificationsViewController.swift]: Nit: you can omit the `options` parameter as it defaults to an empty Dictionary.

File: Classes/Notifications/NoNewNotificationsCell.swift [hunk 2]
```diff
@@ -42,6 +48,23 @@ final class NoNewNotificationsCell: UICollectionViewCell {
             make.top.equalTo(emojiLabel.snp.bottom).offset(Styles.Sizes.tableSectionSpacing)
         }
 
+        reviewGithubAccessButton.setTitle("Review GitHub Access...", for: .normal)
```
Comment (BasThomas) [inline Classes/Notifications/NoNewNotificationsCell.swift]: Should at least be a localized string.

Comment (rnystrom) [reply Classes/Notifications/NoNewNotificationsCell.swift]: Yes! Needs this then g2g I think

Comment (rnystrom) [reply Classes/Notifications/NoNewNotificationsCell.swift]: Also can we omit the “...”?

**Comment by BrianLitwin** [comment]
updated! 👍

**Comment by BrianLitwin** [comment]
If this looks tacky, we could replace with a question mark (or "need help" icon that triggers an alert or ContextMenu that shows Review GitHub Access + help options? We could also put this in the left hand nav bar button's alert controller under "View all", or possibly give it it's own space on the navigation bar.

**Comment by Huddie** [comment]
#2067

**Comment by Huddie** [comment]
If this is a button I feel like blue makes more sense and fits better stylistically with the app (Like load more). Gray has a disabled or side note feel, blue gives off a button feel. IMO

**Comment by rnystrom** [comment]
@BrianLitwin mind resolving conflicts really quick?

**Pull request closed without merging**